pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -677,6 +696,11 @@ public boolean acquire(String tableName) {
       realtimeQuotaOk = tryAcquireToken(realtimeTableName, realtimeTableQueryQuotaEntity);
     }
 
+    if (logicalTableQueryQuotaEntity != null) {
```

Comment:
A separate function is better for readability with a if/else at the beginning of the function.